release-notes/release-notes58.md

```diff
@@ -161,21 +158,28 @@ For information on using premium skins and icon packs, see: [Premium Skins and I
 
 {{site.productname}} 5.8 provides fixes for the following bugs:
 
-- changelog
-
-## Security fixes
-
-{{site.productname}} 5.8 provides fixes for the following security issues:
-
-- changelog
+- Fixed a bug where it wasn't possible to align nested list items.
+- The RGB fields in the color picker dialog were not staying in sync with the color palette and hue slider.
+- The color preview box in the color picker dialog was not correctly displaying the saturation and value of the chosen color.
+- The color picker dialog will now show an alert if it is submitted with an invalid hex color code.
+- Fixed a bug where the `TableModified` event was not fired when adding a table row with the Tab key.
+- Added missing `images_file_types` setting to the exported TypeScript types.
+- Fixed a bug where lists pasted from Word with Roman numeral markers were not displayed correctly. Patch contributed by aautio.
+- The `editor.insertContent` API was incorrectly handling nested `span` elements with matching styles.
+- The HTML5 `small` element could not be removed when clearing text formatting.
+- The Oxide button text transform variable was incorrectly using `capitalize` instead of `none`. Patch contributed by dakur.
+- Fix dialog button text that was using title-style capitalization.
+- Table plugin could perform operations on tables containing the inline editor.
+- Fixed Tab key navigation inside table cells with a ranged selection.
+- The foreground and background toolbar button color indicator is no longer blurry.
+- Fixed a regression in the `tinymce.create()` API that caused issues when multiple objects were created.
+- Fixed the `LineHeight` command causing the `change` event to be fired inconsistently.
 
 ## Deprecated features
 
 The following features have been deprecated with the release of {{site.productname}} 5.8:
 
-- [](#).
-
-### The...
+The undocumented `setIconStroke` Split Toolbar Button API has been deprecated and will be removed in a future release.
```

Comment:
```suggestion
- The undocumented `setIconStroke` Split Toolbar Button API has been deprecated and will be removed in a future release.
```
I believe this is meant to be a list item?

Reply:
One item lists are weird, so no.